desktop/app/index.js

```diff
@@ -85,7 +88,7 @@ if (shouldQuit) {
     windows.forEach(w => {
       w.destroy()
     })
-
-    // TODO: send some event to the service to tell it to shutdown all the things as well
   })
 }
+
+start()
```

Comment:
There was a big if/else (for shouldQuit), when it should short circuit immediately. You can't use `return` (early) from something that's not a function, so I made it a function and called it. You might want to look at the whole file before to grok the change.